crates/sui-framework/sources/governance/validator_set.move

```diff
@@ -439,15 +409,14 @@ module sui::validator_set {
             let v = vector::borrow(vs, i);
             vector::push_back(
                 &mut entries,
-                // Count both self and delegated stake
-                pq::new_entry(validator::gas_price(v), validator::stake_amount(v) + validator::delegate_amount(v))
+                pq::new_entry(validator::gas_price(v), validator::voting_power(v))
             );
             i = i + 1;
         };
         // Build a priority queue that will pop entries with gas price from the highest to the lowest.
         let pq = pq::new(entries);
         let sum = 0;
-        let threshold = self.total_voting_power - self.quorum_threshold;
+        let threshold = voting_power::total_voting_power() - voting_power::quorum_threshold();
```

Comment:
Should we add a `voting_power::threshold()` function?

Reply:
I am not sure if this is a legit thing. Like, this one is just implementation detail